xc/xenops_server_xen.ml

```diff
@@ -2499,13 +2502,15 @@ module VBD = struct
   let epoch_begin task vm disk persistent = match disk with
     | VDI path ->
       let sr, vdi = Storage.get_disk_by_name task path in
-      Storage.epoch_begin task sr vdi persistent
+      let storage_vm = Storage_interface.Vm.of_string("0") in 
```

Comment:
You got a `vm` parameter above, and you could do `vm |> uuid_of_string |> domid_of_uuid ~xc ~xs |> vm_of_domid`.
You'll likely need a `with_xc_and_xs` to get a `xc` and `xs`

Reply:
I wondered howto get the xc and xs, will fix.

Reply:
Unfortunately, the domid_of_uuid won't work in epoch_end, as it has been removed from the xenstore at this point. So I think it is ok to pass an empty string at this point, it will be clear within the storage plugin that we have to fend for ourselves if the string is empty (If we ever decide to use it in this context) 

I am having to rerun the tests as this caused lots to fail :(